gcloud-java-dns/src/main/java/com/google/gcloud/dns/DnsBatch.java

```diff
@@ -150,7 +152,7 @@ public void submit() {
     return ImmutableMap.copyOf(temp);
   }
 
-  private DnsRpc.Callback<ManagedZonesListResponse> listZonesCallback(
+  private DnsRpc.Callback<ManagedZonesListResponse> newListZonesCallback(
```

Comment:
Not sure I like the `new` prefix here and in the following. We might consider `create` if we believe a prefix is needed.